### ISSUE TITLE
Interpolate calibration for changed frequency range

### ISSUE THREAD

**Issue by Kezii** [xaxaxa-dev/vna#23]
Is it realistic to have the way to calibrate across the whole frequency range, with a lot of points, save this calibration, and then scan only a subset of the frequency range for the actual usage?

For example, I can set everything up, start a super big 10 minute calibration, save it, and from that point I can change all the frequency & resolutions I want and it interpolates the calibration 

I don't think that it would matter that much given the relatively limited precision of the instrument

**Comment by turbyho** [comment]
Yes, it's frustrating

**Comment by xaxaxa** [comment]
Yes it should be possible to implement; all that needs to be done is to not clear the cal when frequency range is changed, then add code to interpolate on the fly when displaying values. I would welcome a pull request ;)
